shared/chat/conversation/list-area/index.native.tsx

```diff
@@ -16,6 +16,7 @@ import {ConvoIDContext} from '../messages/ids-context'
 import {FlashList, type ListRenderItemInfo} from '@shopify/flash-list'
 import {getMessageRender} from '../messages/wrapper'
 import {mobileTypingContainerHeight} from '../input-area/normal/typing'
+import {SetRecycleTypeContext} from '../recycle-type-context'
```

Comment:
we keep a ref map in the list that the items can inject extra details about their types into, this lets Message.Type components have more granularity 'text:reactions' etc